Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -200,6 +201,30 @@ public void Exe_PreProc_VersionCheckSucceeds_PreProcSucceeds()
         }
 
         [TestMethod]
+        public void Exe_PostProc_ExecutableNotFound_PostProcFails()
+        {
+            // Arrange
+            string rootDir = TestUtils.CreateTestSpecificFolder(this.TestContext);
+            using (InitializeNonTeamBuildEnvironment(rootDir))
+            {
+                string binDir = CalculateBinDir(rootDir);
+
+                MockBuildAgentUpdater mockUpdater = CreateValidUpdater(binDir, "http://anotherHost");
+                
+                // Act
+                TestLogger logger = CheckExecutionFails(mockUpdater, "end");
+
+                IBootstrapperSettings settings;
+                ArgumentProcessor.TryProcessArgs(new string[] { "end" }, logger, out settings);
+
+                // Assert
+                mockUpdater.AssertUpdateNotAttempted();
+                mockUpdater.AssertVersionNotChecked();
+                logger.AssertErrorLogged(String.Format(Resources.ERROR_PostProcessExeNotFound, settings.PostProcessorFilePath));
```

Comment:
@bgavrilMS There are multiple errors logged, so it is difficult to check the correct one without checking for a specific error, as done here. There is a risk of both incorrect passing and failing from checking the number of errors.